### ISSUE TITLE
Error in text editor

### ISSUE THREAD

**Issue by mhutch** [mono/monodevelop#4393]
Version 7.4 build 985

This error looks like it's written by a WriteLine rather than being reported to the logging system.

May be related to #4392 as it was immediately after it in the log.

```
error while DiffTracker.TrackDocument_TextChanging:System.ArgumentOutOfRangeException: 0 <= length, index(69)+count <= 91
Parameter name: count
Actual value was 31.
  at Mono.TextEditor.Utils.CompressingTreeList`1[T].RemoveRange (System.Int32 index, System.Int32 count) [0x0016a] in /Users/builder/data/lanes/5810/0bed5a33/source/monodevelop/main/src/core/Mono.TextEditor.Shared/Mono.TextEditor.Utils/CompressingTreeList.cs:246 
  at Mono.TextEditor.DiffTracker.TrackDocument_TextChanging (System.Object sender, MonoDevelop.Core.Text.TextChangeEventArgs e) [0x00060] in /Users/builder/data/lanes/5810/0bed5a33/source/monodevelop/main/src/core/Mono.TextEditor.Shared/Mono.TextEditor/Document/DiffTracker.cs:80 
```

> VS bug [#593845](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/593845)

**Comment by mhutch** [comment]
I still see a bunch of writelines in difftracker... we should convert them to logging so they show up in fault telemetry.

**Comment by xamarin-release-manager** [comment]
Fixed in version 7.6.0.557 (master)

Pull Request #4573 merged by: Lluis Sanchez
Author: mono
Commit: 0a9d5590b5e68857ef98b96b3b70ca69c01cbd9b (mono/monodevelop)

**Comment by xamarin-release-manager** [comment]
Fixed in version 7.6.0.557 (master)

Pull Request #4573 merged by: Lluis Sanchez
Author: mono
Commit: 0a9d5590b5e68857ef98b96b3b70ca69c01cbd9b (mono/monodevelop)

**Comment by xamarin-release-manager** [comment]
Fixed in version 8.6.0.802 (master)

Pull Request #4573 merged by: Lluis Sanchez
Author: mono
Commit: 0a9d5590b5e68857ef98b96b3b70ca69c01cbd9b (xamarin/monodevelop)
